apps/admin_api/lib/admin_api/v1/controllers/token_controller.ex

```diff
@@ -89,6 +89,20 @@ defmodule AdminAPI.V1.TokenController do
 
   def stats(conn, _), do: handle_error(conn, :invalid_parameter)
 
+  @spec deploy_erc20(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def deploy_erc20(conn, attrs) do
+    with attrs <- Map.put(attrs, "account_uuid", Account.get_master_account().uuid),
+         {:ok, _} <- authorize(:deploy_erc20, conn.assigns, attrs),
+         {:ok, attrs} <- TokenGate.deploy_erc20(attrs),
```

Comment:
In a future PR, Token will have a status and the gate should handle the original insert, deployment to Ethereum, tracking of the deployment and finally the update of the token to confirmed.